libs/s25main/Window.cpp

```diff
@@ -491,6 +491,19 @@ void Window::DrawLine(DrawPoint pt1, DrawPoint pt2, unsigned short width, unsign
     VIDEODRIVER.GetRenderer()->DrawLine(pt1, pt2, width, color);
 }
 
+void Window::DrawCross(DrawPoint pt, unsigned short length, unsigned short width, unsigned color)
+{
+    auto* renderer = VIDEODRIVER.GetRenderer();
+
+    DrawPoint pt1 = pt - DrawPoint(-length, length);
+    DrawPoint pt2 = pt - DrawPoint(length, -length);
+    DrawPoint pt3 = pt - DrawPoint(length, length);
+    DrawPoint pt4 = pt - DrawPoint(-length, -length);
```

Comment:
I'd avoid the double negation and be const-correct:
```suggestion
    const DrawPoint pt1 = pt - DrawPoint(-length, length);
    const DrawPoint pt2 = pt + DrawPoint(-length, length);
    const DrawPoint pt3 = pt - DrawPoint(length, length);
    const DrawPoint pt4 = pt + DrawPoint(length, length);
```

Reply:
That's just debug code and is going to be removed anyway.